File: src/main/java/org/demoth/aworlds/server2/api/messaging/fromClient/MoveAction.java
package org.demoth.aworlds.server2.api.messaging.fromClient;

import org.demoth.aworlds.server2.api.messaging.Message;

public class MoveAction extends Message {
    public String direction;

    @Override
    public String toString() {
        return "MoveAction{" +
                "direction='" + direction + '\'' +
                '}';
    }
}

File: src/main/kotlin/org/demoth/aworlds/server2/api/messaging/fromClient/MoveAction.kt
package org.demoth.aworlds.server2.api.messaging.fromClient

import org.demoth.aworlds.server2.api.messaging.Message

class MoveAction : Message() {
    var direction: String? = null

    override fun toString(): String {
        return "MoveAction{" +
                "direction='" + direction + '\''.toString() +
                '}'.toString()
    }
}
